src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"gui.securitycraft:scManual.options": "Options disponibles:",
+	"gui.securitycraft:scManual.modules": "Modules disponibles:",
+	"gui.securitycraft:scManual.recipe.keypad": "Placez un cadre et faites un clic droit sur le clavier",
+	"gui.securitycraft:scManual.recipe.keypad_chest": "Placez un Coffre et faites un clic droit sur le clavier.",
+	"gui.securitycraft:scManual.recipe.keypad_furnace": "Placez un Four et faites un clic droit sur le clavier.",
+	"gui.securitycraft:scManual.recipe.reinforced": "Tout bloc blindé peut être fabriqué comme suit, sauf indication contraire dans ce livre. Placez l’équivalent du bloc standard (par exemple Pierre blindée -> Pierre) et cassez-le avec un Blindeur de blocs universel, ou cliquez avec le bouton droit sur le Blindeur de blocs universel, mettez-le dans la fente et fermez l’interface graphique.",
+	"gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Tout bloc blindé peut être fabriqué comme suit, sauf indication contraire dans ce livre. Placez l’équivalent du bloc standard (par exemple Pierre blindée -> Pierre) et cassez-le avec un Blindeur de blocs universel, ou cliquez avec le bouton droit sur le Blindeur de blocs universel, mettez-le dans la fente et fermez l’interface graphique.",
+	"gui.securitycraft:scManual.recipe.adminTool": "L’outil d’administration est uniquement disponible via l’inventaire, s’il est activé via le fichier de configuration.",
+	"gui.securitycraft:scManual.recipe.admin_tool": "L’outil d’administration est uniquement disponible via l’inventaire, s’il est activé via le fichier de configuration..",
+	"gui.securitycraft:scManual.recipe.reinforced_crystal_quartz": "Fabriqué à partir de Bloc de Cristal de Quartz ordinaire dans un Blindeur de blocs universel.",
+	"gui.securitycraft:scManual.recipe.sc_manual": "Le manuel SecurityCraft est uniquement disponible via l’inventaire ou en tapant \"/sc help\" (sans apostrophes).",
+	"gui.securitycraft:scManual.reinforced": "Blocs blindés",
+	"gui.securitycraft:scManual.designedBy": "Conçu par: %s",
+	"gui.securitycraft:ims.hostileAndPlayers": "Mobs hostiles & joueurs",
+	"gui.securitycraft:ims.hostile": "Mobs hostiles",
+	"gui.securitycraft:ims.target": "Cible:",
+	"gui.securitycraft:invScan.emitredstone": "Emettre redstone",
+	"gui.securitycraft:invScan.checkInv": "Vérifier l’inventaire",
+	"gui.securitycraft:invScan.explanation.1": "Ce paramètre vérifiera l’inventaire",
+	"gui.securitycraft:invScan.explanation.2": "d’un joueur, et, s’il contient un",
+	"gui.securitycraft:invScan.explanation.checkInv.3": "élément interdit, l’effaçera",
+	"gui.securitycraft:invScan.explanation.checkInv.4": "de l’inventaire.",
+	"gui.securitycraft:invScan.explanation.emitredstone.3": "élément interdit, émission d’un signal ",
+	"gui.securitycraft:invScan.explanation.emitredstone.4": "redstone pendant 3 secondes.",
```

Comment:
You accidentally changed the casing of the last part of the language keys here. It should be `emitRedstone`, not `emitredstone`.

Reply:
Still needs to be changed.